### PR TITLE
Stats API

### PR DESCRIPTION
**Pull request by douradofrazer**

### CODE REVIEW

**Comment by HenrikHL** [comment]
I don't know if this API is still relevant - I have approved a PR with a correct definition for the Stats API here: https://github.com/dcsaorg/DCSA-Stats-API/blob/master/stats-api-specs.yaml

**Review by HenrikHL — CHANGES_REQUESTED** [review]
Please copy from Stats repositroy master
Please add new endPoint in separate PR

**Comment by douradofrazer** [comment]
> Please copy from Stats repositroy master Please add new endPoint in separate PR

Not gonna recopy it, only the type was a problem. I rechecked your commits on the master repo and realized I missed one commit.
I am checking the error object so updating that.

File: stats/v1/STATS_v1.0.0-Beta-1.yaml [hunk 1]
```diff
@@ -0,0 +1,555 @@
...
+        id: 923k81lod9m4g
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/StatSummary'
+        description: ''
+      parameters:
+        - $ref: '#/components/parameters/api-key'
+      description: This endpoint allows you to report the daily usage statistics of a all DCSA APIs being used.
```
Comment (HenrikHL) [inline stats/v1/STATS_v1.0.0-Beta-1.yaml:240]: There is something wrong with the wording in the end of this description. Maybe the `a` should be removed between `of` and `all`...

File: stats/v1/STATS_v1.0.0-Beta-1.yaml [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+            - `TNT_NTF` (Track and Trace Notifications)
+            - `OVS` (Operational Vessel Schedules)
+            - `EBL` (Electronic Bill of Lading)
+            - `EBL_NTF` (Electronic Bill of Lading Notifications)
+            - `EBL_SUR` (Electronic Bill of Lading Surrender)
+            - `EBL_SUR_RSP` (Electronic Bill of Lading Surrender Response)
+            - `EBL_ISS` (Electronic Bill of Lading Issuance)
+            - `EBL_ISS_RSP` (Electronic Bill of Lading Issuance Response)
+            - `BKG` (Booking)
+            - `BKG` (Booking Notifications)
```
Comment (HenrikHL) [inline stats/v1/STATS_v1.0.0-Beta-1.yaml:429]:
```suggestion
            - `BKG_NTF` (Booking Notifications)
```

File: stats/v1/STATS_v1.0.0-Beta-1.yaml [hunk 3]
```diff
@@ -206,6 +120,22 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                Example 1:
+                  value:
+                    httpMethod: POST
+                    requestUri: /v1/stats
```
Comment (HenrikHL) [inline stats/v1/STATS_v1.0.0-Beta-1.yaml:127]: customized example here is wrong - should be /v1/stats-summary

File: stats/v1/STATS_v1.0.0-Beta-1.yaml [hunk 4]
```diff
@@ -215,6 +145,17 @@ paths:
...
```
Comment (HenrikHL) [inline stats/v1/STATS_v1.0.0-Beta-1.yaml:152]: same

File: stats/v1/STATS_v1.0.0-Beta-1.yaml [hunk 5]
```diff
@@ -224,6 +165,17 @@ paths:
...
```
Comment (HenrikHL) [inline stats/v1/STATS_v1.0.0-Beta-1.yaml:172]: same

File: stats/v1/STATS_v1.0.0-Beta-1.yaml [hunk 6]
```diff
@@ -0,0 +1,639 @@
...
+        id: zksz5ad1zk3eo
+      type: object
+      properties:
+        date:
+          type: string
+          x-stoplight:
+            id: 5vou2ooc47d7y
+          format: date
+          example: '2023-11-01'
+          description: Statistics created date in `ISO 8601` format.
```
Comment (HenrikHL) [inline stats/v1/STATS_v1.0.0-Beta-1.yaml:347]: Is it on purpose that you have this field inside the payload stats-summary array?

**Comment by HenrikHL** [comment]
Stats API is being discontinued here

**Pull request closed without merging**